src/main/java/de/eldoria/jacksonbukkit/JacksonPaper.java

```diff
@@ -53,21 +56,24 @@
  * <ul>
  *     <li>{@link NamespacedKey}
  *     <li>{@link OfflinePlayer}
- *     <li>{@link Inventory} via {@link InventoryWrapper}
+ *     <li>{@link Component}
  * </ul>
  */
 public class JacksonPaper extends JacksonBukkit {
     private final boolean legacyItemStackSerialization;
+    private final Supplier<@Nullable MiniMessage> miniMessage;
 
     /**
      * Create a new JacksonPaper module.
      *
      * @param hexColors                    true to serialize colors as hex by default
      * @param legacyItemStackSerialization true to use spigot based serialization
+     * @param miniMessage supplier for a mini message instance
      */
-    public JacksonPaper(boolean hexColors, boolean legacyItemStackSerialization) {
+    public JacksonPaper(boolean hexColors, boolean legacyItemStackSerialization, Supplier<@Nullable MiniMessage> miniMessage) {
```

Comment:
Not really sure if that results in a CNFE.

Reply:
Open for suggestions xD